src/components/MDX/Challenges/Navigation.tsx

```diff
@@ -65,7 +65,7 @@ export function Navigation({
     if (containerRef.current) {
       const el = containerRef.current;
       el.scrollLeft =
-        challengesNavRef.current[scrollPos].current?.offsetLeft || 0;
+        challengesNavRef.current[scrollPos]?.current?.offsetLeft || 0;
```

Comment:
can you please revert this change. code changes should not be done here otherwise sync from English repo will fail

Reply:
Oh okay. Done, sorry for the inconvenience.